### DISCUSSION TITLE
useGeolocation Vite + Vue 3 Returning Infinity

### DISCUSSION THREAD

**Discussion by michael-evrythngwrx** [vueuse/vueuse#3512] [Q&A] [answered]
Currently implementing useGeolocation(); however, when accessing the value of the return the lat and long go to infinity. In the console when I log coords all data props are OK, but when I try and access the props via coords.value -> the props go to null and lat and long change to infinity. Is there a reason this is happening?

**Comment by BKeanu1989** [comment, marked as answer]
Hope it will be of help to anyone. You just need to use computed on it.
```js
const latNLong = computed(() => {
    if (coords) {
        return [coords.value.latitude, coords.value.longitude]
    }

    return
})
``` 

I just encountered this problem again. Logging via ref shows a value, but directly accessing latitude / longitude didnt work aka Infinity.
